ASReviewLAB.qmd

```diff
@@ -81,71 +82,52 @@ yourself](https://asreview.readthedocs.io/en/latest/lab/installation.html#server
 (or ask your IT-department).
 
 
-*Have you opened ASReview LAB in your browser? If so, you can proceed to
+*Did ASReview LAB open in your browser? If so, you can proceed to
 step 3!*
 
-### Step 3: Creating a project
+## Project Setup
 
-Now that you have installed and opened ASReview LAB, you can create a
-new project. Below you will find a step-by-step guide. Note that the
-screenshots shown below are made in [dark
-mode](https://asreview.readthedocs.io/en/latest/lab/screening.html#dark-mode){target="_blank"}.
+### Step 3: Selecting a Dataset
 
-1.  *New project;*
+Without data, we have nothing to screen. So, you need to tell ELAS
+which dataset you want to screen for relevant articles.
 
-Hover your mouse over the ‘`create`’ button with the plus sign in the
-bottom right corner.
-
-![](images/ASReviewLAB/step_3.png){width=85% fig-align="center"}
-
-2.  *Project name;*
-
-Select Validation Mode, fill out a project name and press ‘`NEXT`’. Note
-that you can fill out your name and a description as well.
-
-![](images/ASReviewLAB/step_3b.png){width=85% fig-align="center"}
-
-For this exercise we are screening in the so-called ‘`Validation Mode`’
+For this exercise we are screening in the so-called `Validation Mode`
```

Comment:
The Validation mode does not exist in version 2

Reply:
Validation mode is still mentioned in the [docs](https://asreview.readthedocs.io/en/latest/lab/validation.html), but the name is slightly changed to validation reviews. Should I use that terminology instead, or do you have another suggestion?